test/suite/test_timestamp07.py

```diff
@@ -142,11 +131,10 @@ def backup_check(self, check_value, valcnt, valcnt2, valcnt3):
         session = self.setUpSessionOpen(conn)
         c = session.open_cursor(self.uri + self.tablename, None)
         c2 = session.open_cursor(self.uri + self.tablename2, None)
-        c3 = session.open_cursor(self.uri + self.tablename3, None)
 
-        # In FLCS the values are bytes, which are numbers, but the tests below are via
-        # string inclusion rather than just equality of values. Not sure why that is, but
-        # I'm going to assume there's a reason for it and not change things. Compensate.
+        # In FLCS the values are bytes, which are numbers, but the tests below are via string
+        # inclusion rather than just equality of values. Not sure why that is, but I'm going
```

Comment:
I suggest getting rid of "I'm going" because the reader has no idea who "I" am. I also realize this wording was in there before this change.

Reply:
Fixed in 7d1a6e037.